specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/amlFilesystems_CreateOrUpdate.json

```diff
@@ -0,0 +1,156 @@
+{
+  "parameters": {
+    "resourceGroupName": "scgroup",
+    "amlFilesystemName": "fs1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-07-23-preview",
+    "amlFilesystem": {
+      "tags": {
+        "Dept": "ContosoAds"
+      },
+      "location": "westus",
+      "properties": {
+        "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+        "encryptionSettings": {
+          "keyEncryptionKey": {
+            "keyUrl": "https://examplekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+            "sourceVault": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/examplekv"
+          }
+        },
+        "maintenanceWindow" : {
+          "dayOfWeek": "Friday",
+          "timeOfDay" : "22:00"
+        },
+        "HSMSettings" : {
+          "container" : "https://somecontainer.blob.core.windows.net/containername",
+          "containerKeyVaultKey" : {
+            "keyUrl" : "https://somekv.vault.azure.net/keys/kvk/3540a47df75541378d3518c6a4bdf5af",
+            "sourceVault" : "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/some-rg/providers/Microsoft.KeyVault/vaults/somekv"
+          },
+          "importPrefix" : "asc",
+          "exportPrefix" : "asc"
+        }
+      },
+      "sku": {
+        "name": "Standard_2G"
```

Comment:
In an example above, this was a sku for HPC cache. I won't call this out everywhere - seems mechanical.

Reply:
Any suggestion on what I could use for a SKU here for amlFilesystems?  I'll need just one for all these examples even if we come up with more at some later time.  That or some place holder that can be replaced later when the real ones are available.

Reply:
Works for me